src/app/audiologist-summary/audiologist-summary.component.html

```diff
@@ -0,0 +1,123 @@
+<div style="text-align: center;">
+  <label class="title">
+    Summary for patient ID: {{patientID}}
+  </label>
+</div>
+
+<div class="margins">
+  <div class="left fifty">
+    <div>
+      <h2>Tinnitus Screener</h2>
+      <p>Type: {{ts}}</p>
+    </div>
+
+    <div>
+      <h2>Tinnitus and Hearing Survey</h2>
+      <p>Section A. Tinnitus: {{thsA}}</p>
+      <p>Section B. Hearing: {{thsB}}</p>
+      <p>Sounds are too loud or uncomfortable: {{thsCtxt}}</p>
+      <p *ngIf="thsC > 0">Examples: {{thsCex}}</p>
+    </div>
+  </div>
+
+  <div class="left fifty">
+    <div>
+      <h2>Tinnitus Functional Index</h2>
+      <p>Overall TFI score: {{overallTFI}}</p>
+    </div>
+
+    <div>
+      <table>
+        <tr>
+          <td>I (INTRUSIVE)</td>
+          <td>{{intrusive}}</td>
```

Comment:
These values sometimes appear as '53.33333333333333'. I don't think we need such accurate decimal precision.

Reply:
Again, we haven't talked about UI, so I didn't see a point in making an arbitrary decision that could be changed later.

Reply:
(Sounds good. We'll make a bug ticket IF we decide this is an undesired feature.)